lab_8_llm/main.py

```diff
@@ -34,6 +29,7 @@ class RawDataImporter(AbstractRawDataImporter):
     """
     A class that imports the HuggingFace dataset.
     """
+    _raw_data: DataFrame
```

Comment:
The variable has already been declared in the parent class `AbstractRawDataImporter`